### PR TITLE
Parrot adjustments.

### PR DESCRIPTION
**Pull request by Belsima**
georg melons and learn to spell "Squawk"

also to_chat

### CODE REVIEW

**Comment by Atermonera** [comment]
Can you give it galcom in has_langs, as well? They're supposed to speak galcom, but in making the animal languages I made them only speak bird, so everything comes out as "chirp chirp caw squawk!"

File: code/modules/mob/living/simple_animal/animals/parrot.dm [hunk 1]
```diff
@@ -643,19 +643,19 @@
...
+		to_chat(usr, "<font color='red'>You have nothing to drop!</font>")
 		return 0
 
 	if(!drop_gently)
 		if(istype(held_item, /obj/item/weapon/grenade))
 			var/obj/item/weapon/grenade/G = held_item
 			G.forceMove(src.loc)
 			G.prime()
-			src << "You let go of the [held_item]!"
+			to_chat(src, "You let go of the [held_item]!"
```
Comment (Atermonera) [inline code/modules/mob/living/simple_animal/animals/parrot.dm]: Missing )

**Comment by Anewbe** [comment]
Compile your code.

**Comment by Belsima** [comment]
Yeah, that's a good idea. Web editor failing me again.

**Pull request merged**